plugin-server/tests/helpers/sql.ts

```diff
@@ -120,7 +120,7 @@ export async function resetTestDatabase(
     await db.end()
 }
 
-export async function insertRow(db: Pool, table: string, objectProvided: Record<string, any>): Promise<void> {
```

Comment:
?

Reply:
We instead use the inferred type rather than defining this explicitly. The signature of the function should be correct.